app/views/index.njk

```diff
@@ -1,89 +1,79 @@
-{% extends 'includes/layout.njk' %}
+{% set pageTitle = "Technical Review Group and Governance" %}
+{% set pageDescription = "This is the lead description needed" %}
 
-{% block outerBody %}
-<section class="nhsuk-hero">
-  <div class="nhsuk-width-container nhsuk-hero--border">
-    <div class="nhsuk-grid-row">
-      <div class="nhsuk-grid-column-two-thirds">
-        <div class="nhsuk-hero__wrapper">
-          <h1 class="nhsuk-u-margin-bottom-3">Technical Review Group and Governance</h1>
-          <p class="nhsuk-body-l nhsuk-u-margin-bottom-0">Helping technical architects and project professionals to navigate our technical governance process.</p>
-        </div>
-      </div>
-    </div>
-  </div>
-</section>
-{% endblock %}
+{% extends 'includes/layout.njk' %}
 
 {% block body %}
 <div class="nhsuk-grid-row">
-  <div class="nhsuk-grid-column-three-quarters">
-<h2 id="how-this-manual-could-help-you">How this manual could help you</h2>
-    <p>As a Technical Architect or Project lead:</p>
-    <ul>
-      <li>You need the most current and relevant information and content to ensure you can deliver effectively</li>
-      <li>You need to understand the correct governance processes you need to follow and who you can speak to for help and support</li>
-      <li>Most importantly you need to be able to see what has already been done; pre-approved standards, policies and principles to form application development and architectural decisions</li>
-    </ul>
-    <p><strong>The result?</strong> You can develop and launch projects faster and more efficiently to improve the lives of patients, clinicians and NHS workers. Ensuring public money doesn’t get wasted through duplicating efforts and creating a documented audit trail of your governance journey.</p>
-    <p>This eBook is designed to help all technical and/or project professionals get to the right information faster through guided navigation. Simply select what you want to achieve either from the homepage or using the side navigation and let the eBook take you to the relevant content.</p>
-    <p>Where available, we will also recommend additional information or resources which might be useful to you and your job.</p>
-    <p>If you have any content you would like to add or update within the eBook, please email your request to <a href="mailto:trg.mailbox@nhs.net">trg.mailbox@nhs.net</a>.
-  </div>
-</div>
+  <div class="nhsuk-grid-column-full app-component-reading-width">
```

Comment:
No styles for `app-component-reading-width` see above comment about app specific styles